docs/tutorial/01-getting-started.md

```diff
@@ -0,0 +1,357 @@
+# Getting Started with Scaffold Stellar
+
+We'll use Scaffold Stellar to create a new Smart Contract project and walk through some simple improvements to show common blockchain interactions that will be helpful when you start crafting your own contracts. This includes:
+
+- storing data
+- authentication
+- handling transactions
+- obfuscating private data
+
+While building these features, you'll also learn the life cycle of smart contract development including
+
+- compiling
+- debugging
+- testing
+- deploying
+- and upgrading
+
+
+## ✅ Prerequisites
+
+You should have a basic understanding of using the command line and of general programming concepts. Stellar Contracts are written in a subset of Rust, although we'll walk through the code together so don't worry if this is your first time with the language.
```

Comment:
Echoing Pam, but also suggesting that we don't say "a subset of" Rust. It's Rust without its standard library, and with some interesting libraries like soroban-sdk instead, but it's still "just Rust."

```suggestion
You should have a basic understanding of using the command line and of general programming concepts. Stellar smart contracts are written in Rust, although we'll walk through the code together so don't worry if this is your first time with the language.
```